emission/core/wrapper/suggestion_sys.py

```diff
@@ -54,6 +54,14 @@
 SEARCH_LIMIT = yelp_auth['search_limit']
 
 
+ZIPCODE_API_KEY = yelp_auth['zip_code_key']
```

Comment:
You need to update the `yelpfusion.json.sample` file to indicate where to get these fields from.
Are you actually using yelp for zipcode -> city mapping?